<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Credit_balance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Credit Balance</string>
   </property>
  </widget>
  <widget class="QLabel" name="Creditperiod">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Credit period</string>
   </property>
  </widget>
  <widget class="QLabel" name="Interesrate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Interest rate %</string>
   </property>
  </widget>
  <widget class="QLabel" name="Typepayment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>181</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Type payment:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Mounthpayment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Mounth payment:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Mounthpayment_score">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AccuredInterest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Accrued interest:</string>
   </property>
  </widget>
  <widget class="QLabel" name="AccuredInterest_score">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TotalPayment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Total payment:</string>
   </property>
  </widget>
  <widget class="QLabel" name="TotalPayment_score">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Balance_Window">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Period_window">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>120.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Procent_Window">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="Credit_Button_Calculate">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Credit_Button_Calculate_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>199</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="Annuities">
      <property name="text">
       <string>Annuities</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="Differentiated">
      <property name="text">
       <string>Differentiated</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="Creditperiod_Mounth">
      <property name="text">
       <string>M</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="Creditperiod_Year">
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Mounthpayment_score_lust">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Mounthpayment_score_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lust differentiated payment: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
